atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -149,8 +149,14 @@ def adjust_for_autorebuild(self):
             self.all_compose_ids = []
 
     def adjust_for_inherit(self):
+        if self.workflow.builder.base_from_scratch:
+            self.log.debug('This is a base image based on scratch. '
+                           'Skipping forward to adjust inheritance for composes.')
```

Comment:
'Skipping forward to adjust inheritance for composes.' =>
'Skipping adjusting composes for inheritance'